tests/test_regrid.py

```diff
@@ -409,11 +412,11 @@ def test_reversed_extract_bounds(self):
         assert north[0], north[-1] == (60, 90)
 
 
+@requires_xesmf
```

Comment:
Add `@requires_xesmf`  decorator on tests that require `xesmf` and skips if `xesmf` is not installed.